terms/allow

```diff
@@ -16,7 +17,6 @@
         {
             "@id": "reproschema:Activity"
         }
-
     ],
     "schema:rangeIncludes": {
         "@id": "schema:Text"
```

Comment:
@satra can you check the range here?

Reply:
i think range would be `schema:Thing` since all the possibilities are terms that are classes.

Reply:
yes true. thanks!